source/beamline/IDEAS/IDEAS13ElementGeDetector.h

```diff
@@ -56,6 +64,21 @@ public slots:
 
 	/// Vortex detectors do not support clearing
 	virtual bool clear() { return false; }
+
+	void setPeakingTime(double time);
+
+	void setPreampGain(double value);
+
+protected:
+	AMReadOnlyPVControl *ge13ElementRealTimeControl_;
+	AMDetector *ge13ElementRealTime_;
```

Comment:
Did you add the real time detector to the class @dleacock?  Typically, we've been putting the one-off detectors inside the beamline class rather than detector classes themselves.